lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -78,7 +78,6 @@ def self.deliver(*args)
     begin
       miq_task&.state_active
       object_name = "#{options[:object_type]}.#{options[:object_id]}"
-      _log.info("Delivering #{ManageIQ::Password.sanitize_string(options[:attrs].inspect)} for object [#{object_name}] with state [#{state}] to Automate")
```

Comment:
I'm not sure about this one - I'm concerned it's going to be logged too late if we do it after `resolve_automation_object`

Reply:
We my want to keep this line as "Attempting to Deliver".  Then later between the create_automation_object and resolve_automation_object, we can try to get the request_id from the uri.